src/components/DataProvider/index.js

```diff
@@ -349,7 +349,7 @@ export default class DataProvider extends Component {
       timeDomain,
       timeSubDomain,
       pointsPerSeries,
-      oldSeries: { data: [], ...seriesObject },
+      oldSeries: { data: [], ...seriesObject, xAccessor },
```

Comment:
Why do you need this? The xAccessor is passed in from the outside, so the outside app already has a reference to it.

Reply:
Yes, you're right. Just found out how to change it from the OpSup side. Reverting this change.